pkg/provisioningv2/prebootstrap/resolve.go

```diff
@@ -62,12 +62,20 @@ func (r *Retriever) GeneratePreBootstrapClusterAgentManifest(controlPlane *rkev1
 		return nil, fmt.Errorf("failed to generate pre-bootstrap cluster-agent manifest: %w", err)
 	}
 
-	return []plan.File{{
-		Content: base64.StdEncoding.EncodeToString(data),
-		Path:    path.Join(capr.GetDistroDataDir(controlPlane), "server/manifests/rancher/cluster-agent.yaml"),
-		Dynamic: true,
-		Minor:   true,
-	}}, nil
+	return []plan.File{
+		{
+			// remove the old manifest file in cluster-agent.yaml, getClusterAgentManifestFile creates a new cluster-agent-job.yaml in 2.12
```

Comment:
I am not 100% sure if we need to add this for prebootstrap manifest, can remove if there's a way to verify.

Reply:
@thatmidwesterncoder